source/column_resizer.ts

```diff
@@ -7,31 +19,23 @@ export interface TableInterface {
   /** Returns the width of the active region. */
   activeWidth: number;
 
-  /** Returns the coordinates of the top left corner
-   *  and bottom right corner.
-   */
-  corners: {
-    topLeft: {
-      x: number 
-      y: number
-    }, 
-    bottomRight: {
-      x: number 
-      y: number
-    }
-  };
-
   /** Returns the width of a column.
    * @param index - The index of the column.
-   * @return The width in pixels of the column at the specified index.
+   * @return The rectange denoting all the corners of the element.
    */
-  getColumnWidth: (index: number) => number;
+  getColumnRect: (index: number) => Rectangle;
 
   /** Resizes the column.
    * @param index - The index of the column.
-   * @param difference The number of pixels to grow or increase the width by.
+   * @param width - The width of the column in pixels.
    */
-  onResize: (columnIndex: number, difference: number) => void;
+  onResize: (columnIndex: number, width: number) => void;
+
+  /** Show the cursor. */
```

Comment:
Rename this, the cursor is always shown anyways. You can rename these methods to ```showResizeCursor``` and ```restoreCursor``` or something else more descriptive.